xds/src/test/java/io/grpc/xds/internal/sds/CommonTlsContextTestsUtil.java

```diff
@@ -18,14 +18,19 @@
 
 import com.google.common.base.Strings;
 import com.google.protobuf.BoolValue;
-import io.envoyproxy.envoy.api.v2.auth.CertificateValidationContext;
-import io.envoyproxy.envoy.api.v2.auth.CommonTlsContext;
-import io.envoyproxy.envoy.api.v2.auth.CommonTlsContext.CombinedCertificateValidationContext;
```

Comment:
I didn't see the correspnding v3 import for CombinedCertificateValidationContext. It will be good to have that

Reply:
Done.